Processing_Sample/CProcessing/Source/CP_Image.c

```diff
@@ -109,16 +109,16 @@ static void CP_Image_DrawInternal(CP_Image img, float x, float y, float w, float
 		break;
 	}
 
-	float a = CP_Math_ClampInt(alpha, 0, 255) / 255.0f;
+	float const a = CP_Math_ClampInt(alpha, 0, 255) / 255.0f;
 
 	// translate and scale image pattern for subimages
 	NVGpaint image = { 0 };
 	if (s0 != s1 && t0 != t1)
 	{
-		float posRatioX = (w / (s1 - s0));
-		float posRatioY = (h / (t1 - t0));
-		float scaleRatioX = (img->w / w) * posRatioX;
-		float scaleRatioY = (img->h / h) * posRatioY;
+		float const posRatioX = (w / (s1 - s0));
```

Comment:
In the future please use the same const style as the creator of the file (west side const). Other than that looks good.

Reply:
I went and changed this around the board for all const changes I did make.